tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/GitHelper.cs

```diff
@@ -1,3 +1,5 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
nit: Just FYI @praveenkuttappan I'm not including the license label in individual test files.